open_city_profile/settings.py

```diff
@@ -68,6 +68,7 @@
     KEYCLOAK_REALM=(str, ""),
     KEYCLOAK_CLIENT_ID=(str, ""),
     KEYCLOAK_CLIENT_SECRET=(str, ""),
+    VERIFIED_PERSONAL_INFORMATION_ACCESS_AMR_LIST=(list, []),
```

Comment:
There is a `docs/config.adoc` file documenting all the settings. This new setting should be added there too.